src/impfbot.py

```diff
@@ -36,13 +37,19 @@ def check_for_slot() -> None:
             log.error("Result is emtpy. (Invalid ZIP Code (PLZ)?)")
         for elem in result:
             if not elem['outOfStock']:
+                local_timezone = timezone(timedelta(hours=1))
```

Comment:
I think it should be 2 hours for CEST/MESZ, 1 hour during the winter

Reply:
thats right